apple/internal/partials/resources.bzl

```diff
@@ -244,6 +272,11 @@ def _resources_partial_impl(
                 elif locale != None:
                     sets.insert(locales_dropped, locale)
                     continue
+            if locales_to_exclude:
+                locale = bundle_paths.locale_for_path(parent_dir)
+                if locale and sets.contains(locales_to_exclude, locale):
```

Comment:
Do you need the `if locale` portion here or can it just be the set check like in the inclusion case?

Reply:
Yes, as `None` is the default return value of `_locales_excluded` and `sets` operations need a non-none value.